doc build: python 3.7  is not installable

carl@x1:~/src/fomu/fomu-workshop/docs$ make env
rm -rf env
./_download/Miniconda3-latest-Linux-x86_64.sh -p /home/carl/src/fomu/fomu-workshop/docs/env -b -f
PREFIX=/home/carl/src/fomu/fomu-workshop/docs/env
Unpacking payload ...
                                                                                                                       
Installing base environment...


Downloading and Extracting Packages:


Downloading and Extracting Packages:

Preparing transaction: done
Executing transaction: done
installation finished.
source env/bin/activate; conda config --system --add envs_dirs /home/carl/src/fomu/fomu-workshop/docs/env/envs
source env/bin/activate; conda config --system --add pkgs_dirs /home/carl/src/fomu/fomu-workshop/docs/_download/pkgs
source env/bin/activate; conda env update --name base --file ./environment.yml
Channels:
 - litex-hub
 - conda-forge
 - defaults
 - timvideos
Platform: linux-64
Collecting package metadata (repodata.json): done
Solving environment: failed
Channels:
 - litex-hub
 - conda-forge
 - defaults
 - timvideos
Platform: linux-64
Collecting package metadata (repodata.json): done
Solving environment: failed

LibMambaUnsatisfiableError: Encountered problems while solving:
  - package menuinst-2.0.1-py311h06a4308_1 requires python >=3.11,<3.12.0a0, but none of the providers can be installed

Could not solve for environment specs
The following packages are incompatible
├─ menuinst is installable with the potential options
│  ├─ menuinst [2.0.0|2.0.1|2.0.2] would require
│  │  └─ python >=3.11,<3.12.0a0 , which can be installed;
│  ├─ menuinst [2.0.0|2.0.1|2.0.2] would require
│  │  ├─ python >=3.10,<3.11.0a0 , which can be installed;
│  │  └─ python_abi 3.10.* *_cp310, which can be installed;
│  ├─ menuinst [2.0.0|2.0.1|2.0.2] would require
│  │  ├─ python >=3.11,<3.12.0a0 , which can be installed;
│  │  └─ python_abi 3.11.* *_cp311, which can be installed;
│  ├─ menuinst [2.0.0|2.0.1|2.0.2] would require
│  │  └─ python_abi 3.12.* *_cp312, which can be installed;
│  ├─ menuinst [2.0.0|2.0.1|2.0.2] would require
│  │  ├─ python >=3.8,<3.9.0a0 , which can be installed;
│  │  └─ python_abi 3.8.* *_cp38, which can be installed;
│  ├─ menuinst 2.0.0 would require
│  │  └─ pypy3.9 >=7.3.12 , which can be installed;
│  ├─ menuinst [2.0.0|2.0.1|2.0.2] would require
│  │  ├─ python >=3.9,<3.10.0a0 , which can be installed;
│  │  └─ python_abi 3.9.* *_cp39, which can be installed;
│  ├─ menuinst [2.0.1|2.0.2] would require
│  │  └─ pypy3.9 >=7.3.13 , which can be installed;
│  ├─ menuinst [2.0.0|2.0.1|2.0.2] would require
│  │  └─ python >=3.10,<3.11.0a0 , which can be installed;
│  ├─ menuinst [2.0.0|2.0.1|2.0.2] would require
│  │  └─ python >=3.12,<3.13.0a0 , which can be installed;
│  ├─ menuinst [2.0.0|2.0.1|2.0.2] would require
│  │  └─ python >=3.8,<3.9.0a0 , which can be installed;
│  └─ menuinst [2.0.0|2.0.1|2.0.2] would require
│     └─ python >=3.9,<3.10.0a0 , which can be installed;
└─ python 3.7**  is not installable because there are no viable options
   ├─ python [3.7.0|3.7.1|...|3.7.9] conflicts with any installable versions previously reported;
   ├─ python [3.7.0|3.7.1|3.7.2|3.7.3|3.7.6] would require
   │  └─ python_abi * *_cp37m, which conflicts with any installable versions previously reported;
   └─ python [3.7.10|3.7.12|...|3.7.9] would require
      └─ python_abi 3.7.* *_cp37m, which conflicts with any installable versions previously reported.

make: *** [Makefile:34: env] Error 1
carl@x1:~/src/fomu/fomu-workshop/docs$

--- a/docs/environment.yml
+++ b/docs/environment.yml
@@ -4,7 +4,7 @@ channels:
 - conda-forge
 - defaults
 dependencies:
-- python=3.7
+- python
 - pip
 - pycairo
 - pango

